audit/projects/k8s-artifacts-prod/buckets/k8s-artifacts-prod-vuln-dashboard/bucketpolicyonly.txt

```diff
@@ -1,4 +0,0 @@
-Bucket Policy Only setting for gs://k8s-artifacts-prod-vuln-dashboard:
```

Comment:
This is the expected outcome of https://github.com/kubernetes/k8s.io/issues/1642